apps/src/codebridge/PopUpButton/PopUpButton.tsx

```diff
@@ -130,6 +130,22 @@ export const PopUpButton = ({
         createPortal(
           <FocusTrap
             focusTrapOptions={{
+              isKeyForward: event => {
+                if (event.key === 'ArrowDown') {
+                  event.stopPropagation();
+                  return true;
+                }
+                // If we remove this line, tab will move focus but focus will
+                // not be trapped. Same with shift+tab below.
+                return event.key === 'Tab';
+              },
+              isKeyBackward: event => {
+                if (event.key === 'ArrowUp') {
+                  event.stopPropagation();
+                  return true;
+                }
+                return event.key === 'Tab' && event.shiftKey;
+              },
```

Comment:
Nice! I pulled your branch and confirmed the up/down arrow keys work to navigate among popup button options.

Noticing that when a user hits 'Escape' now when the options are displayed, the focus goes back to top of screen instead of returning to the popup button.
In the 'fallback-event' div, could you add a check for 'Escape' key also?

```
<div
  id="fallback-element"
  ...
  onKeyDown={event => {
    if (event.key === 'Enter' || event.key === ' ' || event.key === 'Escape') {
      event.preventDefault();
      setIsOpen(false);
      if (buttonRef) {
        buttonRef.focus();
      }
    }
  }}
  ...
```

Reply:
Yes I could add the escape key, but that doesn't change the behavior. The + button dropdown returns focus to the button after close (which already works on escape) but the files/folders don't work either way. Unless you've gotten that to work locally, which I'd be excited to see!

Reply:
From my testing the escape functionality hasn't changed with this PR, but I have added a ticket to fix this issue we've had with popupButtons! https://codedotorg.atlassian.net/browse/SL-1235

Reply:
Ahh...I think when I was testing locally, my mouse must have been hovered over ellipsis and I didn't realize it. Thanks!